Week5/solutionweek5.txt

```diff
@@ -0,0 +1,94 @@
+import csv
+
+#create and write the records in csv file
+with open("c://sqlite3//student_csv.csv","w",newline="")as stud:

```

Comment:
This Statment is wrong because,this is forward slash( / ).

you please rewrite this statment ..